editor/src/components/canvas/controls/size-box.tsx

```diff
@@ -162,45 +193,101 @@ class ResizeEdge extends React.Component<ResizeEdgeProps> {
       baseTop +
       (this.props.direction === 'vertical' ? -this.props.visualSize.height / 2 : -lineSize / 2)
 
+    const isEdgeDragged =
+      this.props.dragState != null &&
+      this.props.dragState.type === 'RESIZE_DRAG_STATE' &&
+      this.props.dragState.start != null &&
+      this.props.dragState.edgePosition.x === this.props.position.x &&
+      this.props.dragState.edgePosition.y === this.props.position.y
+
     return (
-      <div
-        ref={this.reference}
-        style={{
-          pointerEvents: 'initial',
-          position: 'absolute',
-          left: left,
-          top: top,
-          width: width,
-          height: height,
-          boxSizing: 'border-box',
-          backgroundColor: 'transparent',
-          cursor: this.props.resizeStatus === 'enabled' ? this.props.cursor : undefined,
-        }}
-      />
+      <React.Fragment>
+        <div
+          ref={this.reference}
+          onMouseOver={() => this.setState({ showLabel: true })}
+          onMouseOut={() => this.setState({ showLabel: false })}
+          style={{
+            pointerEvents: 'initial',
+            position: 'absolute',
+            left: left,
+            top: top,
+            width: width,
+            height: height,
+            boxSizing: 'border-box',
+            backgroundColor: 'transparent',
+            cursor: this.props.resizeStatus === 'enabled' ? this.props.cursor : undefined,
+          }}
+        />
+        {(this.state.showLabel || isEdgeDragged) && (
```

Comment:
`when(`

Reply:
Fixed in 5b68dd64a689f218fbe4e203d4db7b5a9433a636.